.scalafmt.conf

```diff
@@ -20,11 +20,6 @@ newlines {
   sometimesBeforeColonInMethodReturnType = true
 }
 
-verticalMultiline {
-  newlineAfterImplicitKW = true
-  newlineBeforeImplicitKW = true
-}
```

Comment:
This should have been replaced by
```
verticalMultiline.atDefnSite = true
newlines.implicitParamListModifierForce = [before, after]
```
But this create a massive reformatting. Should we put it in ?

Reply:
yeah let's do it!

is `verticalMultiline.atDefnSite = true` the right replacement though? The release notes only mention [newlines.implicitParamListModifierForce](https://github.com/scalameta/scalafmt/releases/tag/v3.4.0)

Reply:
I got it from the example in the scalafmt [doc](https://scalameta.org/scalafmt/docs/configuration.html#vertical-multiline-with-implicit-parameter-lists)

Reply:
Not putting `verticalMultiline.atDefnSite = true` will have only the implicits parameters defined on new lines.
I think the example is also using `verticalMultiline.atDefnSite = true` so all params are defined in a consistent way. 
Having only `newlines.implicitParamListModifierForce` creates a reasonable changeset

Reply:
tbf we should actually just remove this as the default behavior with the latest versions seems to be inline with the previous properties. Setting this new one just makes things weird

Reply:
will revert last commit